### DISCUSSION TITLE
[UI] Add BTCPay / custom logo in menu bar

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#2427] [Ideas / Feature Requests]
It is already possible to add a custom logo in the checkout experience of a store.

It would be nice if the top menu bar would contain by default the BTCPay logo, and optionally a custom logo of the user.

Really not priority / important at all, but this feature is implemented in Nextcloud, and really helps to make the local server feel "special", I like it.

This might even be possible already with a server CSS theme, no clue...

**Comment by bolatovumar** [comment]
@MaxHillebrand you can achieve this by uploading a custom theme to your BTCPay and in that theme you should be able to set the styles for a custom logo. It's not a straightforward way of doing but it's a way. Not sure if there is an easier way.

I guess we _could_ augment the "Server settings: Theme" page to allow specifying a custom logo which would make the process straightforward for the user.

**Reply by ghost** [reply]
Thanks for confirming that it's possible with a custom CSS.

Maybe a Theme plugin with all this jazz, and removing the current settings out of the main package :eyes: :wink:

**Comment by dstrukt** [comment]
Dennis and I have discussed improving the theming. from easier color selections and granular variable control to logos, etc.. once we improve the foundation.

Taken note of this, and should hopefully be addressed in time.

Agreed it's a nice feature that you should configured once in a theme easily, and it's propagated in the relevant places.
